packages/nextra-theme-docs/src/components/toc.tsx

```diff
@@ -98,9 +111,9 @@ export function TOC({ toc, filePath }: TOCProps): ReactElement {
       {hasMetaInfo && (
         <div
           className={cn(
-            hasHeadings && 'nextra-toc-footer _mt-8 _pt-8',
-            '_sticky _bottom-0 _flex _flex-col _items-start _gap-2 _pb-8',
-            '_-mx-1 _px-1' // to hide focused toc links
+            hasHeadings && 'nextra-toc-footer _pt-4',
+            '_flex _flex-col _items-start _gap-2 _pb-4',
```

Comment:
reduced padding-y to match footer in sidebar
<img width="332" alt="image" src="https://github.com/user-attachments/assets/13769700-328f-49f9-bae1-0c734d94e1d0">

<img width="345" alt="image" src="https://github.com/user-attachments/assets/c016ef42-b45e-474e-9c32-eca46b4286f1">

Reply:
or maybe leave as current `_pt-6`?

Reply:
I’m good with either.